src/replication/DealReplicationWorker.ts

```diff
@@ -109,7 +102,7 @@ export default class DealReplicationWorker extends BaseService {
             })]);
         } else {
           this.logger.debug(`No cron, execute immediately`);
-          await this.replicate(newReplicationWork);
+          this.replicate(newReplicationWork); // no await so that this can return quickly, enabling parallel polling
```

Comment:
the error will not be caught below if this is not awaited. You probably need to put try catch logic inside  this.replicate.